ui/design/src/components/Avatar/index.tsx

```diff
@@ -18,6 +18,7 @@ export interface AvatarProps extends CommonInterface<HTMLDivElement> {
   faded?: boolean;
   size?: AvatarSize;
   publicImgPath?: string;
+  borderColor?: AvatarBorderColor;
```

Comment:
this is confusing, I would expect to be able to pass a color here like `borderColor: blue`

Reply:
the color codes weren't defined yet in the theme, so I added them. I can refactor to use blue and accent colors instead

Reply:
the issue is with the naming.
if i present to you a component that accepts a property `borderColor` what you would expect it to be?

Reply:
the name of the color as string